### ISSUE TITLE
AdaTarget on Real SR images

### ISSUE THREAD

**Issue by Magauiya** [yhjo09/AdaTarget#1]
Dear authors,

I would like to thank you for sharing the code. Considering that for real images, kernels are different, your method should perfectly fit to that task.

Thus, I have tried your code on real images from AIM 2020 real super-resolution challenge and got worse results compared to original GT pre-trained network. I have used yours ATG pretrained ckpt & haven`t changed anything except network (also of-the-shelf SR model) and dataset.

_As far as I understand there are no experiments with real dataset, aren`t they? If it was conducted, can you (dis)confirm that adatarget does not work for real scenarios?_ 

Thank you!

**Comment by yhjo09** [comment]
Hi,
I also expect that our method has a benefit to a real world scenario and a
real world SR dataset.
Regarding your question, I conducted experiments on the DRealSR dataset and
I found there is a minor performance degradation.
However, with some modifications (use the lower error one between the
original target patch and the adaptive target patch) I got a slightly
better result, but there is still no significance to the performance gap.
We need to research more in this direction in the future.

Here are the results of the DRealSR test set.
Method:RRDB PSNR:32.01 SSIM:0.8867 LPIPS:0.3094
Method:Ours PSNR:31.96 SSIM:0.8867 LPIPS:0.3104
Method:Ours+ PSNR:32.02 SSIM:0.8868 LPIPS:0.3074

Thanks.
